package.json

```diff
@@ -29,6 +35,7 @@
   },
   "scripts": {
     "babelBuild": "babel src -d dist/node",
+    "lint": "eslint .",
```

Comment:
no, good catch. @ttacon you should make a `"ci"` script that calls `yarn test && yarn lint && yarn check`